src/main/webapp/i18n/de/complaint.json

```diff
@@ -31,8 +31,8 @@
             "listOfComplaints": {
                 "exercise": "Übung",
                 "submissionId": "Einreichungs-ID",
-                "reviewerLogin": "Reviewerlogin",
-                "reviewerName": "Reviewername",
+                "reviewerLogin": "Login Reviewer:in",
```

Comment:
The usage of "Reviewer:in" vs. "Tutor:innen" is not consistent.

I would suggest to use "Tutor:innen" or "Gutachter:innen" as translation of "Reviewer" at any place.